### PR TITLE
Add ember-cli-babel as a dependency and clean up

### LINKED ISSUE #10: Bump faker to 3.1.0
Currently version seems to be `"Faker": "~2.1.0"` which is missing `count` for `faker.random.alphaNumberic` :(

### LINKED ISSUE #11: Deprecation on ember-cli 2.12
When used with `ember-cli@2.12.0`, `ember-faker` emits the following deprecation warning:

> DEPRECATION: Addon files were detected in `/path/to/app/node_modules/ember-faker/addon`, but no JavaScript preprocessors were found for `ember-faker`. Please make sure to add a preprocessor (most likely `ember-cli-babel`) to in `dependencies` (NOT `devDependencies`) in `ember-faker`'s `package.json`.

### PR DESCRIPTION
**Pull request by YoranBrondsema**
* Fixes #11 
* Fixes #10 
* Modernizes the project by updating dependencies
* Adds a test

### CODE REVIEW

**Comment by YoranBrondsema** [comment]
Tests pass locally but no on Travis :(. @johnotander I don't think I can clear the cache on Travis CI. Would you mind doing that? I have a feeling it's due to that.

File: bower.json [hunk 1]
```diff
@@ -1,18 +1,6 @@
...
-    "ember-resolver": "~0.1.7",
-    "loader.js": "stefanpenner/loader.js#1.0.1",
-    "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "rwjblue/ember-cli-test-loader#0.0.4",
-    "ember-load-initializers": "stefanpenner/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.1.8",
-    "ember-qunit-notifications": "0.0.4",
-    "qunit": "~1.15.0",
-    "Faker": "~2.1.0"
+    "Faker": "4.1.0"
```
Comment (alanlalonde) [inline bower.json]: Faker is available as an npm package, removing all need to keep using Bower.

Comment (YoranBrondsema) [inline bower.json]: You're right. I will look into making this change this weekend.

Comment (YoranBrondsema) [inline bower.json]: @alanlalonde I tried importing Faker as an NPM module but unfortunately it's only available as a CommonJS module. I couldn't really find an easy way of importing it, in the same way the Bower component is imported (the NPM module does not export a built file). So I'd like to leave it as it is for now, and we can look into importing it as a NPM module in a future PR. In any case, this PR does quite some much needed modernizing!

**Comment by YoranBrondsema** [comment]
@johnotander Any updates?

**Comment by Subtletree** [comment]
[ember-browserify](https://github.com/ef4/ember-browserify#using-ember-browserify-in-addons) is an option for importing faker from npm

**Comment by Subtletree** [comment]
Actually ember-cli-mirage import `faker` from npm using [ember-cli-node-assets](https://github.com/dfreeman/ember-cli-node-assets).

https://github.com/samselikoff/ember-cli-mirage/blob/master/index.js

**Comment by YoranBrondsema** [comment]
@Subtletree Perfect, thanks! I still want to merge this PR first and move away from Bower in a next PR.

@johnotander in case you're not actively using this addon yourself anymore, would you mind giving contributor permissions to others so that we can continue maintaining the addon? I think there are still a few projects depending on it! You could start with me so I can get this modernization PR merged :-)

**Comment by johnotander** [comment]
Hey @YoranBrondsema, thanks for volunteering and sorry for the delay in responding. I've sent you an invite to become a collaborator so if you wanted to take over the maintenance of this project I'd be delighted 💓 . I've unfortunately been short on time for maintaining some of my libraries lately.

**Comment by johnotander** [comment]
Also, if you send along your npm username I can add you as an owner of the package so you can push new versions.

**Comment by YoranBrondsema** [comment]
@johnotander No problem, thanks! My NPM username is _yoranbrondsema_

**Comment by johnotander** [comment]
Added you as an owner to the npm package ❤️

**Pull request merged**